packages/asset-swapper/src/utils/alt_mm_implementation_utils.ts

```diff
@@ -12,8 +12,11 @@ import {
     AltQuoteRequestData,
     AltQuoteSide,
     AltRfqtMakerAssetOfferings,
+    LogFunction,
 } from '../types';
 
+const SUCCESS_CODE = 201;
```

Comment:
Would be nice if our `StatusCodes` package was more robust...

Reply:
Great idea, isn't this code in our `StatusCodes` package?

Reply:
I don't think it is

Reply:
It's not